src/from_geo.rs

```diff
@@ -50,13 +42,20 @@ impl<'a, T: Borrow<Point<f64>>> TryInto<GGeom<'a>> for &'a [T] {
     }
 }
 
-impl<'a> TryInto<GGeom<'a>> for &'a LineString<f64> {
-    type Err = Error;
-
-    fn try_into(self) -> Result<GGeom<'a>, Self::Err> {
-        let coord_seq = create_coord_seq_from_vec(self.0.as_slice())?;
-
-        GGeom::create_line_string(coord_seq)
+impl<'a> TryFrom<&'a LineString<f64>> for GGeom<'a> {
+    type Error = Error;
+
+    fn try_from(linestring: &'a LineString<f64>) -> Result<GGeom<'a>, Self::Error> {
+        let mut coords = CoordSeq::new(linestring.num_coords() as u32, CoordDimensions::TwoD)?;
```

Comment:
I'm wondering: why changing the implementation?

Reply:
Because 

> Library authors should usually not directly implement this trait, but should prefer implementing the `TryFrom` trait, which offers greater flexibility and provides an equivalent `TryInto` implementation for free, [...] [TryInto](https://doc.rust-lang.org/std/convert/trait.TryInto.html)

Is that OK with you?

Reply:
No I meant about the content of the function itself. I'm happy about using the std trait directly.

Reply:
Using `self.0` implies a certain implementation of `LineString`, whereas using `points_iter()` relies on the interface, which seems more robust.

Reply:
I can easily revert that change if you prefer. And I think I relied on the `self.0` method for MultiPolygon.

Reply:
Ok, makes sense!